tox.ini

```diff
@@ -108,7 +108,7 @@ commands =
 [molecule_common]
 changedir = {toxinidir}
 deps =
-    ansible-lint==4.2.0
```

Comment:
Should we be more conservative and keep ansible-lint locked at a newer, known good version? (that would be 4.3.5 I suppose).